Psmdb 1376 release notes 7.0.4

nastena1606 opened the pull request.

nastena1606 committed:
PSMDB-1376 Release notes 7.0.3

modified:   .github/workflows/main.yml
	modified:   docs/404.md
	new file:   docs/release_notes/7.0.3-2.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   requirements.txt
	modified:   variables.yml

nastena1606 committed:
Added 7.0.4 release notes

igorsol reviewed (approved):
LGTM

sandraromanchenko commented on mkdocs-base.yml:
```diff
@@ -117,7 +118,7 @@ plugins:
           output_path: '_pdf/PerconaServerforMongoDB-7.0.pdf'
           cover_title: 'Server for MongoDB Documentation 7.0'
-          cover_subtitle: 7.0.2-1 (October 5, 2023)
+          cover_subtitle: 7.0.3-2 (December 11, 2023)
```
```suggestion
          cover_subtitle: 7.0.4-2 (December 11, 2023)
```

sandraromanchenko commented on docs/release_notes/index.md:
```diff
@@ -1,5 +1,6 @@
 
 # Percona Server for MongoDB 7.0 release notes
 
+* [Percona Server for MongoDB 7.0.4-2 (2023-12-11)](7.0.3-2.md)
```
```suggestion
* [Percona Server for MongoDB 7.0.4-2 (2023-12-11)](7.0.4-2.md)
```

kwilktx commented:
@nastena1606 @igorsol @sandraromanchenko 

I think we need to mention the Downgrade and fCV changes and the Backward-Incompatible Features - It is too big a thing not to mention - with consequences

Starting in MongoDB 7.0:
    Binary downgrades are no longer supported for MongoDB Community Edition.
    
    The [setFeatureCompatibilityVersion](https://www.mongodb.com/docs/v7.0/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) command requires an additional parameter, confirm, which must be set to true to upgrade or downgrade fCV.
    
    https://www.mongodb.com/docs/v7.0/release-notes/7.0/#downgrade-considerations
    
https://www.mongodb.com/docs/v7.0/release-notes/7.0-compatibility/#std-label-7.0-downgrade-incompatible

wildcard indexes, config servers that store data, columnar indexes that would have to be removed.

Other than that the bugs that you mention as fixed look good.

nastena1606 committed:
Update mkdocs-base.yml

Co-authored-by: Sandra Romanchenko <53295797+sandraromanchenko@users.noreply.github.com>

nastena1606 committed:
Update docs/release_notes/index.md

Co-authored-by: Sandra Romanchenko <53295797+sandraromanchenko@users.noreply.github.com>

nastena1606 committed:
Added downgrade considerations

nastena1606 committed:
Added warning about 80203

modified:   docs/release_notes/7.0.2-1.md
	modified:   docs/release_notes/7.0.4-2.md

sandraromanchenko commented on docs/release_notes/7.0.4-2.md:
```diff
@@ -0,0 +1,51 @@
+    - After you have upgraded the database, allow your deployment to run without setting [setFeatureCompatibilityVersion](https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) to 7.0 for a burn-in period to ensure the likelihood of downgrade is minimal. When you are confident that the likelihood of downgrade i ...
+    - The [setFeatureCompatibilityVersion](https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/#mongodb-dbcommand-dbcmd.setFeatureCompatibilityVersion) command requires an additional parameter, `confirm`. You must set it to `true` to upgrade or downgrade fCV.
+
+* Due to [time series sharded cluster operation routing issue](https://jira.mongodb.org/browse/SERVER-80203) occurred in MongoDB version 7.0.1 - 7.0.3 and fixed in version 7.0.4, follow closely the upstream remediation recommendations to identify and recover orphanated collections.
```
```suggestion
* Due to [time series sharded cluster operation routing issue](https://jira.mongodb.org/browse/SERVER-80203) occurred in MongoDB version 7.0.1 - 7.0.2 and fixed in version 7.0.3, follow closely the upstream remediation recommendations to identify and recover orphanated collections.
```

nastena1606 committed:
Added GA for AWS IAM
